e2e/favoritePage_test.go

```
@@ -0,0 +1,199 @@
+
+	reAdded := false
+	for _, page := range reFavResp.Data {
+		if page.Pathname == pagePath1 && page.Favorite {
+			reAdded = true
+		}
+	}
+	assert.True(t, reAdded, "Re-favorited page should appear as active")
+
+	// Cleanup: remove both test pages
+	for _, path := range []string{pagePath1, pagePath2} {
+		client.POST(APIBasePath+"/favorite-pages", map[string]interface{}{
```

golangci-lint (2.12.2)

[error] Error return value of `client.POST` is not checked

(errcheck)